.drone.yml

```diff
@@ -10,27 +10,27 @@ steps:
 
   # use minimum supported rust version for most steps
   - name: prepare repo
-    image: clux/muslrust:1.64.0
+    image: alpine:3
```

Comment:
Does this really have any benefit? Anyway it has to download the muslrust image later, so downloading an extra image here will only take extra time and space.

Reply:
Yep, namely that we'll never have to upgrade it, in the same way we have to change all the rust versions when there's an upgrade. By consolidating all the cargo related steps into one image, we really only have to use that image for one step, and upgrade only that one.

Reply:
Upgrading is a simple replace all so thats not a good argument imo.